appeals/web/src/server/appeals/appeal-details/interested-party-comments/view-and-review/__tests__/__snapshots__/view-and-review.test.js.snap

```diff
@@ -102,7 +102,7 @@ exports[`interested-party-comments GET /review-comment with data should render r
                     </div>
                     <div class="govuk-summary-list__row"><dt class="govuk-summary-list__key"> Address</dt>
                         <dd class="govuk-summary-list__value">Not provided</dd>
-                        <dd class="govuk-summary-list__actions"><a class="govuk-link" href="/appeals-service/appeal-details/2/interested-party-comments/3670/edit/address?review=true"> Add<span class="govuk-visually-hidden"> address</span></a>
+                        <dd class="govuk-summary-list__actions"><a class="govuk-link" href="/appeals-service/appeal-details/2/interested-party-comments/3670/edit/address?review=true&amp;editAddress=false"> Add<span class="govuk-visually-hidden"> address</span></a>
```

Comment:
Not sure if it's an issue but the `&` seems to be escaped/encoded as `&amp;`

Reply:
Will fix forward in my currently overlapping bug ticket